src/internal/mapping.rs

```diff
@@ -196,7 +196,7 @@ impl<K: ArenaId, V: serde::Serialize> serde::Serialize for Mapping<K, V> {
         self.chunks
             .iter()
             .flatten()
-            .take(self.max())
+            .take(self.max() + 1)
```

Comment:
Nice find! This in itself might be worth a PR!

Reply:
If you prefer, I can cherry pick the commit onto a new branch and submit a new PR?

Reply:
That would be great!